l10n_br_nfe_spec/models/v4_00/leiauteNFe.py

```diff
@@ -0,0 +1,5120 @@
+# Copyright 2020 Akretion - Raphael Valyi <raphael.valyi@akretion.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+# Generated Sat Oct 10 03:37:29 2020 by https://github.com/akretion/generateds-odoo
+# and generateDS.py.
+# Python 3.8.2 (default, Apr 27 2020, 15:53:34)  [GCC 9.3.0]
+#
+import textwrap
+from odoo import fields, models
+
+# Código de Regime Tributário.
+# Este campo será obrigatoriamente preenchido com:
+CRT_EMIT = [
```

Comment:
An improvement that could be made in the future is split the constants code and class objects